test/test_torch.py

```diff
@@ -8426,6 +8426,89 @@ def test_numpy_index(self):
             self.assertFalse(isinstance(idx, int))
             self.assertEqual(x[idx], x[int(idx)])
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
```

Comment:
So... I suppose this test is a reasonable smoketest, but what I *really* want tested, is that now a Torch tensor can successfully be used by the numba library. I don't see that tested here, and that doesn't seem good!

Reply:
You should test if numba is available before running said test, because not all test environments will have numba available (CI for one).

Reply:
👍 Sounds great to me, I'd also prefer to have this tested on the pytorch-side.

I will port a few internal tests that test this integration and handle the numba library detection in a manner similar to how scipy is currently detected.

Reply:
I've pushed a set of baseline numba integration cases which are "working on my machine" 😉.

This will likely require adding numba to your CI configuration, presumably at pietern/pytorch-dockerfiles? Would you like to coordinate a change to your CI before this pull?

Reply:
You can test out adding it yourself by conda install'ing numba in `.jenkins/pytorch/test.sh`. Do it in only one configuration, e.g., one of our conda CUDA ones.